frontend/src/router/index.ts

```diff
@@ -47,7 +47,7 @@ const routes = [
         component: LandingView,
     },
     {
-        path: routeItems.dashboard.path,
+        path: routeItems.dashboard.url,
```

Comment:
I am not sure I understand why changing from the key name from ".path" to ".url" matter in this breadcrumb issue.

1. To me, the key name shouldn't be "**url**" from the "**route**Items", since this is for **Vue Router**; and Vue Router is correct to use "**path**" as the key name, so even we supply from the routerItems object for the value, should still call it "**path**".

2. **Route's** "**path**" isn't a "**url**", path and url are different. **URL** contains **many parts** and is a link that you are able to access with it while "**path**" is from within the Route and is part of the URL. Mixing them isn't a good idea.

Reply:
Hello, Ian! The options were discussed at a past scrum. To use any other key name other than "url" we would need to use the templating syntax and that would break the style, The 'url' key name is expected by the primevue Breadcrumb component,
https://github.com/bcgov/nr-forests-access-management/pull/1169#issuecomment-1924803550

Reply:
Jota, I wasn't so sure what you were discussing last time so didn't comment on that. Primevue provides flexible options, including "url" (for different purpose), it does not mean we should use it to make app work (and I suspect we used the wrong option initially before this issue). If fixing style is expected, then we should fix styling. We should discuss again after scrum. 
@MCatherine1994

Reply:
Hi Jota, I had a quick chat with Ian to have more understanding of this breadcrumb component. So we feel it's better to follow their example. Cause if we change to use "url", it feels like we mixed the internal link vs external link. "route" is for internal routes, and "url" is for external links. Even it works fine for now, but long term we might feel confused again. So let's switch to follow this example and fix our style, thanks so much and sorry for the confusion!!

<img width="931" alt="image" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/ed26153b-9bd5-49a8-a2fa-6bce824096e2">

Reply:
I'll make the change to template.